### PR TITLE
[OSDOCS-6403]: Add warning on cluster deletion docs

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13
Issue:

https://issues.redhat.com/browse/OSDOCS-6403
Link to docs preview:

https://62221--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-deleting-cluster.html

https://62221--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.html
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-deleting-cluster.adoc [hunk 1]
```diff
@@ -25,14 +25,18 @@ You can delete a {product-title} (ROSA) with AWS Security Token Service (STS) cl
...
 ====
 The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
 ====
 
 If add-ons are installed, the cluster deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
 endif::sts[]
 
+[IMPORTANT]
+====
+If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagset key/value pair of the installer-created resources and labeled with a value of `owned` will be deleted as well.
```
Comment (tmalove) [inline modules/rosa-deleting-cluster.adoc:38]:
```suggestion
If the cluster that created the VPC during the installation is deleted, the associated installation program-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagset key-value pair of the resources created by the installation program and labeled with a value of `owned` will also be deleted.
```

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
Hi @mletalie , I made a couple of suggestions otherwise, it looks good. Also, I could not find any guidance on 'tagset' that made sense in this context, so you might want to check with someone if this is a term we should use. Thanks!

**Comment by mletalie** [comment]
image
tagSet reference. Determinging if capitalization is necessary.

**Comment by mletalie** [comment]
> Hi @mletalie , I made a couple of suggestions otherwise, it looks good. Also, I could not find any guidance on 'tagset' that made sense in this context, so you might want to check with someone if this is a term we should use. Thanks!

Hi @tmalove, thanks! I did reach back out to the SRE and he did confirm we should use "tagset" for reference. However, (as I found as well)  we should change case to "tagSet". Thanks!

**Comment by mletalie** [comment]
> New changes are detected. LGTM label has been removed.

Removed the change that prompted the label removal (specifically the "code block" format on "tagSet")

File: modules/rosa-deleting-cluster.adoc [hunk 2]
```diff
@@ -25,14 +25,18 @@ You can delete a {product-title} (ROSA) with AWS Security Token Service (STS) cl
...
 ====
 The cluster deletion must complete before you remove the IAM resources, because the resources are used in the cluster deletion and clean-up processes.
 ====
 
 If add-ons are installed, the cluster deletion takes longer because add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
 endif::sts[]
 
+[IMPORTANT]
+====
+If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same tagSet key-value pair of the installer-created resources and labeled with a value of `owned` will also be deleted.
```
Comment (JoeAldinger) [inline modules/rosa-deleting-cluster.adoc:38]:
```suggestion
If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC will also be deleted, resulting in the failure of all the clusters that are using the same VPC. Additionally, any resources created with the same `tagSet` key-value pair of the installer-created resources and labeled with a value of `owned` will also be deleted.
```

Comment (JoeAldinger) [reply modules/rosa-deleting-cluster.adoc:38]: If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.

**Review by JoeAldinger — COMMENTED** [review]
Two small things and we are set to merge.

File: modules/rosa-deleting-cluster.adoc (hunk 2 above)
Comment (mletalie) [reply modules/rosa-deleting-cluster.adoc:38]: > If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.
Hello @JoeAldinger,
    Thanks for taking a look at this doc. I will check out the style guide and let you know if I have any questions about what is stated. FWIW, I had actually pulled that text from an existing doc. Regardless, let me take a look at the style guide and make sure I understand it. Thanks!  
image

Comment (mletalie) [reply modules/rosa-deleting-cluster.adoc:38]: > If this is a specific object reference it needs to be in back ticks or if it is a generic it needs to be tag set. I agree with @tmalove and think `installer-created` needs to be addressed per [redhat-documentation.github.io/supplementary-style-guide/#_i](https://redhat-documentation.github.io/supplementary-style-guide/#_i) please.

...and yes, going back over my notes with the SRE, it is specific and should have back ticks. Will update.

Comment (mletalie) [reply modules/rosa-deleting-cluster.adoc]: Hello @JoeAldinger.
     I believe all the requested changes to be completed in order to merge are now complete. Thanks!

**Comment by JoeAldinger** [comment]
@mletalie do you want this just in `main` and `4.13` or should this go into `4.14` too.

**Comment by mletalie** [comment]
> @mletalie do you want this just in `main` and `4.13` or should this go into `4.14` too?

Hello @JoeAldinger, I am sorry about the delayed response. I should have put 4.13+.....not sure if you can still put it in 4.14 as well? My apologies, thank you!

**Pull request merged**